cmd/nerdctl/run_linux.go

```diff
@@ -136,17 +136,6 @@ func setPlatformOptions(
 		opts = append(opts, oci.WithDevShmSize(shmBytes/1024))
 	}
 
-	pid, err := cmd.Flags().GetString("pid")
-	if err != nil {
-		return nil, err
-	}
-	pidOpts, pidInternalLabel, err := generatePIDOpts(ctx, client, pid)
-	if err != nil {
-		return nil, err
-	}
-	internalLabels.pidContainer = pidInternalLabel
-	opts = append(opts, pidOpts...)
-
```

Comment:
The pidns changes seem irrelevant to utsns, could you consider splitting the commit? Not necessary to open a separate PR.

Reply:
Sure, will try and get to today

Reply:
@AkihiroSuda done